### PR TITLE
#20 update versions

### PR DESCRIPTION
**Pull request by anton6tak**

### CODE REVIEW

File: libraries.json [hunk 1]
```diff
@@ -337,12 +332,12 @@
...
     "github": "Kotlin/kotlinx.atomicfu",
     "category": "Utils",
-    "maven": "https://kotlin.bintray.com/kotlinx/org/jetbrains/kotlinx/atomicfu-native/"
+    "maven": "https://repo1.maven.org/maven2/org/jetbrains/kotlinx/atomicfu-native/"
   },
   {
     "github": "Dominaezzz/kgl",
     "category": "UI",
-    "maven": "https://dl.bintray.com/dominaezzz/kotlin-native/com/kgl/kgl-core/"
+    "maven": "https://maven.pkg.github.com/Dominaezzz/kgl"
```
Comment (Alex009) [inline libraries.json:340]: authorization required. we can't fetch info from github packages

File: libraries.json [hunk 2]
```diff
@@ -387,7 +382,7 @@
   {
     "github": "apollographql/apollo-android",
     "category": "Network",
-    "maven": "https://dl.bintray.com/apollographql/android/com/apollographql/apollo/apollo-api/"
+    "maven": "https://repo1.maven.org/maven2/com/apollographql/apollo/apollo-android-support/"
```
Comment (Alex009) [inline libraries.json:385]: it's not multiplatform artifact. but https://repo1.maven.org/maven2/com/apollographql/apollo/apollo-api/ is multiplatform

File: libraries.json [hunk 3]
```diff
@@ -427,17 +422,17 @@
   {
     "github": "robstoll/atrium",
     "category": "Test",
-    "maven": "https://dl.bintray.com/robstoll/tutteli-jars/ch/tutteli/atrium/atrium-fluent-en_GB/"
+    "maven": "https://repo1.maven.org/maven2/dev/inmo/tgbotapi/"
```
Comment (Alex009) [inline libraries.json:425]: it's another library at all

File: libraries.json [hunk 4]
```diff
@@ -427,17 +422,17 @@
...
-    "github": "robstoll/atrium",
+    "github": "robstoll/atrium", 
     "category": "Test",
     "maven": "https://dl.bintray.com/robstoll/tutteli-jars/ch/tutteli/atrium/atrium-fluent-en_GB-android/"
   },
   {
     "github": "robstoll/atrium",
     "category": "Test",
-    "maven": "https://dl.bintray.com/robstoll/tutteli-jars/ch/tutteli/atrium/atrium-fluent-en_GB-android-js/"
+    "maven": "https://repo1.maven.org/maven2/dev/inmo/tgbotapi-js/"
```
Comment (Alex009) [inline libraries.json:435]: it's another library at all

File: libraries.json [hunk 5]
```diff
@@ -507,17 +502,19 @@
...
-    "maven": "https://dl.bintray.com/nephyproject/stable/blue/starry/jsonkt/"
+    "maven": "https://repo1.maven.org/maven2/blue/starry/jsonkt/"
   },
   {
     "github": "p7s1-ctf/SevenFacette",
     "category": "Test",
-    "maven": "https://dl.bintray.com/p7s1qa/sevenfacette-jvm/de/p7s1/qa/sevenfacette/core/"
+    "maven": "https://maven.pkg.jetbrains.space/sevenfacette/p/sevenfacette/sevenfacette",
+    "username": "85dcdf4c-d6cd-486f-85d3-654904e1cf22",
+    "password": "365171aab3c77e6aa2016b73c29ba738280dd96e94dddfc15e45ef65ffbcec5b"
```
Comment (Alex009) [inline libraries.json:517]: what is it? where it's used? why it's added?

File: libraries.json [hunk 6]
```diff
@@ -542,17 +539,17 @@
...
     "github": "Liftric/Auth",
     "category": "Integration",
-    "maven": "https://dl.bintray.com/liftric/maven/com/liftric/auth/"
+    "maven": "https://repo1.maven.org/maven2/com/liftric/auth/"
   },
   {
     "github": "ACINQ/bitcoink",
     "category": "Integration",
-    "maven": "https://dl.bintray.com/acinq/libs/fr/acinq/bitcoink/bitcoink/"
+    "maven": "https://repo1.maven.org/maven2/fr/acinq/bitcoin/"
```
Comment (Alex009) [inline libraries.json:552]: at this path no library at all.
but here https://repo1.maven.org/maven2/fr/acinq/bitcoin/bitcoin-kmp/ has

File: libraries.json [hunk 7]
```diff
@@ -572,22 +569,22 @@
   {
     "github": "chRyNaN/chords",
     "category": "UI",
-    "maven": "https://dl.bintray.com/chrynan/chrynan/com/chrynan/chords/chords-core/"
+    "maven": "https://repo.repsy.io/mvn/chrynan/public/com/chrynan/chords/"
```
Comment (Alex009) [inline libraries.json:572]: here no library at all.
should be https://repo.repsy.io/mvn/chrynan/public/com/chrynan/chords/chords-core/

File: libraries.json [hunk 8]
```diff
@@ -572,22 +569,22 @@
...
   },
   {
     "github": "chRyNaN/inject",
     "category": "DI",
-    "maven": "https://dl.bintray.com/chrynan/chrynan/com/chrynan/inject-common/"
+    "maven": "https://repo.repsy.io/mvn/chrynan/public/com/chrynan/inject/"
```
Comment (Alex009) [inline libraries.json:577]: https://repo.repsy.io/mvn/chrynan/public/com/chrynan/inject/inject/

File: libraries.json [hunk 9]
```diff
@@ -572,22 +569,22 @@
...
   },
   {
     "github": "chRyNaN/mapper",
     "category": "Utils",
-    "maven": "https://dl.bintray.com/chrynan/chrynan/com/chrynan/mapper-core/"
+    "maven": "https://repo.repsy.io/mvn/chrynan/public/com/chrynan/mapper/"
```
Comment (Alex009) [inline libraries.json:582]: https://repo.repsy.io/mvn/chrynan/public/com/chrynan/mapper/mapper-core/

File: libraries.json [hunk 10]
```diff
@@ -572,22 +569,22 @@
...
   },
   {
     "github": "chRyNaN/logger",
     "category": "Logging",
-    "maven": "https://dl.bintray.com/chrynan/chrynan/com/chrynan/logger/logger-core/"
+    "maven": "https://repo.repsy.io/mvn/chrynan/public/com/chrynan/logger/"
```
Comment (Alex009) [inline libraries.json:587]: https://repo.repsy.io/mvn/chrynan/public/com/chrynan/logger/logger-core/

File: libraries.json [hunk 11]
```diff
@@ -602,27 +599,27 @@
...
     "github": "fluidsonic/fluid-locale",
     "category": "Device",
-    "maven": "https://dl.bintray.com/fluidsonic/kotlin/io/fluidsonic/locale/fluid-locale/"
+    "maven": "https://repo1.maven.org/maven2/io/fluidsonic/locale/fluid-locale/"
   },
   {
     "github": "fluidsonic/fluid-currency",
     "category": "Utils",
-    "maven": "https://dl.bintray.com/fluidsonic/kotlin/io/fluidsonic/currency/fluid-currency/"
+    "maven": "https://repo1.maven.org/maven2/io/fluidsonic/currency/"
```
Comment (Alex009) [inline libraries.json:607]: https://repo1.maven.org/maven2/io/fluidsonic/currency/fluid-currency/

File: libraries.json [hunk 12]
```diff
@@ -602,27 +599,27 @@
...
   },
   {
     "github": "fluidsonic/fluid-i18n",
     "category": "Utils",
-    "maven": "https://dl.bintray.com/fluidsonic/kotlin/io/fluidsonic/i18n/fluid-i18n/"
+    "maven": "https://repo1.maven.org/maven2/io/fluidsonic/i18n/"
```
Comment (Alex009) [inline libraries.json:612]: https://repo1.maven.org/maven2/io/fluidsonic/i18n/fluid-i18n/

File: libraries.json [hunk 13]
```diff
@@ -782,7 +779,7 @@
   {
     "github": "splendo/kaluga",
     "category": "UI",
-    "maven": "https://dl.bintray.com/kaluga/com.splendo.kaluga/com/splendo/kaluga/alerts/"
+    "maven": "https://dl.bintray.com/kaluga/com.splendo.kaluga"
```
Comment (Alex009) [inline libraries.json:782]: why this change in this way?

**Review by Alex009 — CHANGES_REQUESTED** [review]

File: libraries.json (hunk 1 above)
Comment (anton6tak) [reply libraries.json:340]: How is better to set info about that? https://github.com/Dominaezzz/kgl#usage

File: libraries.json (hunk 5 above)
Comment (anton6tak) [reply libraries.json:517]: This is credentials to get from private maven repo https://github.com/munichbughunter/SevenFacette#jetbrains-space

File: libraries.json (hunk 1 above)
Comment (Alex009) [reply libraries.json:340]: remove this lib. automatic fetch can't auth

File: libraries.json (hunk 5 above)
Comment (Alex009) [reply libraries.json:517]: 1. fetcher not use this fields
2. it's secrets that should not be here
3. fetcher not have auth logic - remove this lib

File: libraries.json (hunk 2 above)
Comment (anton6tak) [reply libraries.json]: fixed

File: libraries.json (hunk 3 above)
Comment (anton6tak) [reply libraries.json]: fixed

File: libraries.json (hunk 1 above)
Comment (anton6tak) [reply libraries.json:340]: removed

File: libraries.json (hunk 4 above)
Comment (anton6tak) [reply libraries.json]: fixed

File: libraries.json (hunk 5 above)
Comment (anton6tak) [reply libraries.json:517]: removed

File: libraries.json (hunk 9 above)
Comment (anton6tak) [reply libraries.json]: fixed

File: libraries.json (hunk 10 above)
Comment (anton6tak) [reply libraries.json]: fixed

File: libraries.json (hunk 11 above)
Comment (anton6tak) [reply libraries.json]: fixed

File: libraries.json (hunk 12 above)
Comment (anton6tak) [reply libraries.json]: fixed

File: libraries.json (hunk 13 above)
Comment (anton6tak) [reply libraries.json]: reverted changes

File: libraries.json [hunk 14]
```diff
@@ -142,27 +137,27 @@
   {
     "github": "korlibs/klogger",
     "category": "Logging",
-    "maven": "https://dl.bintray.com/korlibs/korlibs/com/soywiz/korlibs/klogger/klogger/"
+    "maven": "https://repo1.maven.org/maven2/com/soywiz/korlibs/klogger/"
```
[the remark this answers is not in the document]
Comment (anton6tak) [reply libraries.json:140]: fixed

File: libraries.json [hunk 15]
```diff
@@ -142,27 +137,27 @@
...
   },
   {
     "github": "korlibs/kbignum",
     "category": "Math",
-    "maven": "https://dl.bintray.com/korlibs/korlibs/com/soywiz/korlibs/kbignum/kbignum/"
+    "maven": "https://repo1.maven.org/maven2/com/soywiz/korlibs/kbignum/"
```
[the remark this answers is not in the document]
Comment (anton6tak) [reply libraries.json:145]: fixed

File: libraries.json [hunk 16]
```diff
@@ -142,27 +137,27 @@
...
   },
   {
     "github": "korlibs/krypto",
     "category": "Cryptography",
-    "maven": "https://dl.bintray.com/korlibs/korlibs/com/soywiz/korlibs/krypto/krypto/"
+    "maven": "https://repo1.maven.org/maven2/com/soywiz/korlibs/krypto/"
```
[the remark this answers is not in the document]
Comment (anton6tak) [reply libraries.json:150]: fixed

File: libraries.json [hunk 17]
```diff
@@ -142,27 +137,27 @@
...
   },
   {
     "github": "korlibs/korim",
     "category": "Device",
-    "maven": "https://dl.bintray.com/korlibs/korlibs/com/soywiz/korlibs/korim/korim/"
+    "maven": "https://repo1.maven.org/maven2/com/soywiz/korlibs/korim/"
```
[the remark this answers is not in the document]
Comment (anton6tak) [reply libraries.json:155]: fixed

File: libraries.json [hunk 18]
```diff
@@ -142,27 +137,27 @@
...
   },
   {
     "github": "korlibs/korau",
     "category": "Device",
-    "maven": "https://dl.bintray.com/korlibs/korlibs/com/soywiz/korlibs/korau/korau/"
+    "maven": "https://repo1.maven.org/maven2/com/soywiz/korlibs/korau/"
```
[the remark this answers is not in the document]
Comment (anton6tak) [reply libraries.json:160]: fixed

File: libraries.json [hunk 19]
```diff
@@ -297,7 +292,7 @@
   {
     "github": "AAkira/Napier",
     "category": "Logging",
-    "maven": "https://dl.bintray.com/aakira/maven/com/github/aakira/napier/"
+    "maven": "https://repo1.maven.org/maven2/io/github/aakira/napier"
```
[the remark this answers is not in the document]
Comment (anton6tak) [reply libraries.json:295]: fixed

File: libraries.json [hunk 20]
```diff
@@ -427,22 +417,22 @@
   {
     "github": "robstoll/atrium",
     "category": "Test",
-    "maven": "https://dl.bintray.com/robstoll/tutteli-jars/ch/tutteli/atrium/atrium-fluent-en_GB/"
+    "maven": "https://repo1.maven.org/maven2/dev/ch/tutteli/atrium/atrium-core-robstoll-lib"
```
[the remark this answers is not in the document]
Comment (anton6tak) [reply libraries.json:420]: corrected to "https://repo1.maven.org/maven2/dev/inmo/tgbotapi/"

File: libraries.json [hunk 21]
```diff
@@ -717,7 +702,7 @@
   {
     "github": "copper-leaf/clog",
     "category": "Logging",
-    "maven": "https://dl.bintray.com/copper-leaf/oss/io/copper-leaf/clog/"
+    "maven": "https://repo1.maven.org/maven2/io/github/copper-leaf/clog-core"
```
[the remark this answers is not in the document]
Comment (anton6tak) [reply libraries.json:705]: fixed

**Review by Alex009 — APPROVED** [review]
